### DISCUSSION TITLE
Using Melodyne for tuning?

### DISCUSSION THREAD

**Discussion by krutoi2000** [stakira/OpenUtau#1369] [Q&A]
Can I use Melodyne to extract pitch of voice and place it in Utau?

**Comment by oxygen-dioxide** [comment]
What you want is a pitch analyzer. We aren't able to use melodyne, but we already have crepe, an open source machine learning pitch analyzer for frq generation of voicebank audio file. We can make it support analyzing pitch from audio part to voice part.
